src/hooks/useMultiSelect.ts

```diff
@@ -4,68 +4,54 @@ import {
   SelectedItems,
 } from "../components/MultiSelect/MultiSelect";
 
-interface useMultiSelectProps {
-  multiSelectId: string;
-  items: MultiSelectItem[];
-}
-
 /**
- * The useMultiSelect The hook returns an object containing all the props and state needed to handle the selectedItems.
+ * The useMultiSelect The hook returns an object containing all the props and state needed to handle the selectedItems in conjunction with one single Multiselect component as well as a Group of MultiSelects.
```

Comment:
Nitpick: these long lines are hard to read because of the scrollbar. We try to keep lines to 80 characters max. If possible, can you return on the long sentences?

Reply:
Fixed? I am not sure I know what "return on long sentences" means but I added a line break.